dartsim/src/EntityManagement_TEST.cc

```diff
@@ -201,6 +204,23 @@ TEST(EntityManagement_TEST, ConstructEmptyWorld)
   EXPECT_NEAR(meshShapeScaledSize[0], 0.2553, 1e-4);
   EXPECT_NEAR(meshShapeScaledSize[1], 0.3831, 1e-4);
   EXPECT_NEAR(meshShapeScaledSize[2], 0.0489, 1e-4);
+
+  auto heightmapLink = model->ConstructEmptyLink("heightmap_link");
+  heightmapLink->AttachFixedJoint(child, "heightmap_joint");
+
+  auto heightmapFilename = ignition::common::joinPaths(
+      IGNITION_PHYSICS_RESOURCE_DIR, "heightmap_bowl.png");
+  ignition::common::ImageHeightmap data;
+  EXPECT_GE(0, data.Load(heightmapFilename));
```

Comment:
it looks like this expectation is checking that the return value is less than or equal to 0, but [ImageHeightmap::Load](https://github.com/ignitionrobotics/ign-common/blob/ignition-common4_4.1.0/graphics/src/ImageHeightmap.cc#L34-L37) returns `-1` for failures and `0` for success. I think it should be `EXPECT_EQ(0, data.Load(...));`

Reply:
e700b5b